### ISSUE TITLE
Get stats about the external services

### ISSUE THREAD

**Issue by Khady** [onitu/onitu#92]
We can ask for local stats about a driver with a call to the route `/api/v1.0/entries/<name>/stats` and get a few informations.

```
{
    "info": {
        "age": 260.1202609539032,
        "cpu": 0.0,
        "create_time": 1411678512.858972,
        "ctime": "0:04.50",
        "mem": 0.5,
        "mem_info1": "22M",
        "mem_info2": "2G",
        "started": 1411678512.864391
    },
    "name": "hubic-B",
    "status": "ok",
    "time": 1411678772.989669
}
```

I would like to have more stats, not about the entry itself, but about the remote service. Like how many free gb are available. The informations wouldn't be the same for every drivers but it shouldn't be a problem.

**Comment by k4nar** [comment]
Yeah, we should add a handler for asking a service how much free space it has. Maybe we could also get other information, but I don't see what right now.
